<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python focus stacking</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="image_preview">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>861</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../HighResImages/focus_stacked.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loaded_images_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Loaded images (0)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="loaded_images_table_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>251</width>
      <height>321</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1153</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="load_images_action"/>
    <addaction name="save_project_action"/>
    <addaction name="export_output_action"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="process_images_menu">
    <property name="title">
     <string>process images</string>
    </property>
    <addaction name="align_images_action"/>
    <addaction name="stack_images_action"/>
    <addaction name="align_and_stack_images_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="process_images_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="load_images_action">
   <property name="text">
    <string>Load images</string>
   </property>
   <property name="toolTip">
    <string>load images for stacking (ctrl + l)</string>
   </property>
   <property name="statusTip">
    <string>load images for stacking (ctrl + l)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="save_project_action">
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="toolTip">
    <string>save project to disk (ctrl + s)</string>
   </property>
   <property name="statusTip">
    <string>save project to disk (ctrl + s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="export_output_action">
   <property name="text">
    <string>Export output</string>
   </property>
   <property name="iconText">
    <string>export output image (ctrl + e)</string>
   </property>
   <property name="toolTip">
    <string>export output image (ctrl + e)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Exit the program</string>
   </property>
   <property name="whatsThis">
    <string>Exit the program</string>
   </property>
  </action>
  <action name="align_images_action">
   <property name="text">
    <string>Align images</string>
   </property>
  </action>
  <action name="stack_images_action">
   <property name="text">
    <string>Stack images</string>
   </property>
  </action>
  <action name="align_and_stack_images_action">
   <property name="text">
    <string>Align and stack images</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
